ci_cd/utils/file_io.py

```diff
@@ -0,0 +1,21 @@
+"""Utilities for handling IO operations."""
+import re
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:  # pragma: no cover
+    from pathlib import Path
+    from typing import Optional, Tuple
+
+
+def update_file(
+    filename: "Path", sub_line: "Tuple[str, str]", strip: "Optional[str]" = None
+) -> None:
+    """Utility function for tasks to read, update, and write files"""
+    if strip is None and filename.suffix == ".md":
+        # Keep special white space endings for markdown files
+        strip = "\n"
```

Comment:
consider adding a test for this

Reply:
Indeed. Tests should be added mainly for the `ci_cd/utils/versions.py` file as well. I'll make an issue of it.